client/src/pages/aboutUsPage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "../stylings/home.css";
+import Footer from "../components/footer";
+import AboutUs from '../components/aboutUs';
+
+// import AboutUs from "../components/aboutUsV2";
```

Comment:
Whichever one you end up going with, make sure to remove code references relevant to the other